v4/export/ir_impl.go

```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"strconv"
+	"github.com/pkg/errors"
```

Comment:
1. Use `"github.com/pingcap/errors"`
2. Put this after `"go.uber.org/zap"`

Reply:
done